config/src/main/kotlin/maru.config/Config.kt

```diff
@@ -38,29 +38,11 @@ data class FollowersConfig(
 data class P2P(
   val ipAddress: String,
   val port: String,
+  val discoveryPort: String,
```

Comment:
I think that discovery has to be toggleable so that users enable/disable it at will. Also, it would make sense to disable it in all the current MaruApp and other tests, and enable it for the new tests only

Reply:
done